app/views/play/CampaignView.coffee

```diff
@@ -1242,29 +1242,31 @@ module.exports = class CampaignView extends RootView
     @userPollsRecord = @supermodel.loadModel(@userPollsRecord, null, 0).model
     onRecordSync.call @ if @userPollsRecord.loaded
 
-  loadPoll: ->
-    url = "/db/poll/#{@userPollsRecord.id}/next"
-    @poll = new Poll().setURL url
+  loadPoll: (url, forceShowPoll=false) ->
+    url ?= "/db/poll/#{@userPollsRecord.id}/next"
+    tempLoadingPoll = new Poll().setURL url
```

Comment:
Why are we using `tempLoadingPoll` instead of @poll?

Reply:
`@poll` is referenced in the modal. Therefore, when the user clicks "Next" button in the modal if `@poll` is mutated it breaks the modal. Therefore we only want to assign the poll to `@poll` once it is loaded and we're showing the next modal.

It reduces race conditions.